libphobos/libdruntime/gcc/emutls.d

```diff
@@ -0,0 +1,316 @@
+// GNU D Compiler emulated TLS routines.
+// Copyright (C) 2011-2019 Free Software Foundation, Inc.
```

Comment:
By the way, the copyright year should be 2019 only?

Reply:
A copy/paste leftover. Should be fixed in the patch I just sent to the ML.